index.html

```diff
@@ -2999,32 +2999,56 @@ <h3>Trust Model</h3>
         <ul>
           <li>
 The [=verifier=] trusts the [=issuer=] to issue the [=credential=] that
-it received. To establish this trust, a [=credential=] is expected to either:
+it received after verification has been performed. 
+To establish this trust, a [=credential=] is expected to either:
             <ul>
               <li>
-Secure the [=credential=] with a <a href="#securing-mechanisms">securing mechanism</a> establishing that the
+Be secured with a <a href="#securing-mechanisms">securing mechanism</a> establishing that the
 [=issuer=] generated the [=credential=] (that is, it is a
 [=verifiable credential=]), or
               </li>
               <li>
 Have been transmitted in a way clearly establishing that the [=issuer=]
-generated the [=verifiable credential=] and that the
-[=verifiable credential=] was not tampered with in transit or storage. This
+generated the [=credential=] and that the
+[=credential=] was not tampered with in transit or storage. This
 trust could be weakened depending on the risk assessment of the [=verifier=].
```

Comment:
That is a question that should be addressed to the WG, since it is not addressed in this PR

Reply:
Let me clearer — Why did you delete `verifiable` from these two instances of `[=verifiable credential=]`, so they are now just `[=credential=]`?

Reply:
Because the opening paragraph talks about credential, viz:

> The verifier trusts the issuer to issue the credential that it received. To establish this trust, a credential is expected to either:

Consequently the two bullet points that follow it must talk about the credential.